src/qcodes/instrument_drivers/stahl/stahl.py

```diff
@@ -144,14 +147,32 @@ class Stahl(VisaInstrument):
 
     Args:
         name
-        address: A serial port address
+        address: A serial port address or TCP-IP visa address
```

Comment:
For completeness could you document name and kwargs too.

name: Name of the instrument
**kwargs: forwarded to baseclass

Reply:
will do